DOCSP-49867: Add JsonReader example (#636)
mongodb/docs-csharp: source/data-formats/extended-json.txt

      { "_id" : { "$oid" : "573a1391f29313caabcd9637" }, "createdAt" : { "$date" : "1970-01-19T12:51:39.609Z" }, "numViews" : 36520312 }

> Alternatively, you can use the ``JsonReader`` class to read Extended JSON documents. The
> following example reads uses the ``JsonReader`` class to read the same Extended JSON document
> as the preceding example into a ``BsonDocument`` object:

Reviewer: S: combine into one sentence as its repetitive:
```suggestion
Alternatively, you can use the ``JsonReader`` class to read Extended JSON documents
into other formats such as ``BsonDocument``, as shown in the following example:
```

- `JsonWriter <{+new-api-root+}/MongoDB.Bson/MongoDB.Bson.IO.JsonWriter.html>`__
- `JsonReader <{+new-api-root+}/MongoDB.Bson/MongoDB.Bson.IO.JsonReader.html>`__
- `JsonWriterSettings <{+new-api-root+}/MongoDB.Bson/MongoDB.Bson.IO.JsonWriterSettings.html>`__

> - `JsonReaderSettings <{+new-api-root+}/MongoDB.Bson/MongoDB.Bson.IO.JsonReaderSettings.html>`__

Author: According to Rishit and Boris, this isn't worth discussing.
